pinot-core/src/main/java/org/apache/pinot/core/query/aggregation/function/DistinctAggregationFunction.java

```diff
@@ -70,6 +81,35 @@ public AggregationResultHolder createAggregationResultHolder() {
     return new ObjectAggregationResultHolder();
   }
 
+  private ColumnDataType[] fieldSpecTypeToColumnTypes() {
+    ColumnDataType[] columnDataTypes = new ColumnDataType[_dataTypes.length];
+    for (int i = 0; i < _dataTypes.length; i++) {
```

Comment:
extract `_dataTypes.length` to local variable?

Reply:
Done